tensorflow/python/distribute/input_lib_test.py

```diff
@@ -1194,5 +1194,45 @@ def dataset_fn(ctx):
           strategy,
           sess=sess)
 
+class DistributedIteratorPerReplicaTest(test.TestCase, parameterized.TestCase):
+
+  @combinations.generate(
+      combinations.combine(
+          mode=["eager"],
+          distribution=[
+              strategy_combinations.mirrored_strategy_with_two_gpus]))
+  def testInputSignatureForPerReplicaValues(self, distribution):
+    with distribution.scope():
+      def dataset_fn(input_context):
+        return tf.data.Dataset.from_tensor_slices(np.zeros([4, 4])).apply(tf.data.experimental.prefetch_to_device('gpu'))
```

Comment:
which "gpu" is it going to prefetch to?

I think probably better to remove this from here since this will not generalize to the CPU tests i mentioned above

Reply:
addressed, same as https://github.com/tensorflow/tensorflow/pull/38968#discussion_r489158187